README.md

```diff
@@ -113,14 +106,6 @@ To run all tests, including unit and integration tests, run the following comman
   rails test:all
 ```
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the above command to your tests to the correct one i.e. `rspec spec/`  instead of   `rails test:all `. this ensure that your documentation looks professionnal with correct and appropriate information content.